piptools/scripts/compile.py

```diff
@@ -33,11 +34,24 @@
 )
 from ..writer import OutputWriter
 
+ALL_BUILD_DISTRIBUTIONS = ("editable", "sdist", "wheel")
```

Comment:
Are these supposed to be ordered? If not, it's a set.

Reply:
Good point.